requirements/scenarios.md

```diff
@@ -282,7 +282,7 @@ Maybe this is an internally created artifact with a known signing key.
 This key may be distributed using a trusted third party mechanism.
 
 1. The user obtains a trusted key for a particular artifact.
-1. The user downloads and verifies the artifact using Notary v2 and their known key.
+1. The user downloads and verifies the artifact using Notary Project specification implementation and their known key.
```

Comment:
The user downloads and verifies the artifact using the Notary Project specification implementation

Reply:
I will update it to `implementations of the Notary Project specification`